### PR TITLE
Generate Financial Report - Exam Type Filtering

### PR DESCRIPTION
**Pull request by akroon3r**
Further development on the generate CSV modal required exam type filtering to be completed. Based on the exam type elected via radio button, a url parameter is passed to the GET method used to generate the final csv. Based on the parameter provided, dynamic filtering is done to the exam object created via query.

### CODE REVIEW

File: frontend/src/exams/generate-financial-report-modal.vue [hunk 1]
```diff
@@ -71,11 +71,21 @@
...
+            }else if (this.selectedExamType === 'veterinary') {
+              exam_type = 2;
+            }else if (this.selectedExamType === 'milk_tank') {
+              exam_type = 3;
+            }
+
             let url = '/exams/export/?start_date=' + form_start_date + '&end_date=' + form_end_date + '&exam_type='
                       + exam_type
+
+            console.log("URL: ", url)
```
Comment (sjrumsby) [inline frontend/src/exams/generate-financial-report-modal.vue]: Superfluous console.

File: frontend/src/exams/generate-financial-report-modal.vue [hunk 2]
```diff
@@ -56,8 +56,8 @@
             endDate: '',
             options: [
               {text: 'ITA - Individual and Group ', value: 'ita'},
-              {text: 'Pesticide', value: 'pesticide'},
-              {text: 'Bulk Milk Tank Grader', value: 'milk_tank'}
+              {text: 'Veterinary Exam', value: 'veterinary'},
```
Comment (sjrumsby) [inline frontend/src/exams/generate-financial-report-modal.vue]: We will need to be able to search for pesticide exams, I think. Do you think there should be an "other" as well, that filters everything not in the above?

File: frontend/src/exams/generate-financial-report-modal.vue (hunk 1 above)
Comment (akroon3r) [reply frontend/src/exams/generate-financial-report-modal.vue]: fixed

File: frontend/src/exams/generate-financial-report-modal.vue (hunk 2 above)
Comment (akroon3r) [reply frontend/src/exams/generate-financial-report-modal.vue]: pushed up

**Pull request closed without merging**